Sources/Remote Logging/Crash Logging/SentryEventJSException.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import Sentry
+
+public struct JSException {
+    public let type: String
+    public let value: String
```

Comment:
Could this property be named as `message`?

Reply:
Sure, I used `value` to match Sentry exception parameters ([reference](https://develop.sentry.dev/sdk/event-payloads/exception/#attributes)) but I'm happy to rename it.

Reply:
Suggestion applied in https://github.com/WordPress/gutenberg/pull/59221/commits/363cb123ba416622269c45acaf5e8df23a9151e9, https://github.com/wordpress-mobile/WordPress-iOS/pull/22655/commits/037244d34e1ec231ac66add2acfce615b0772572  and https://github.com/Automattic/Automattic-Tracks-iOS/pull/278/commits/07a7012a5af7a5e8c47bc5959e5d1331a8cffe6f.